scanner/indexer/indexer.go

```diff
@@ -19,6 +20,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/remote/transport"
 	"github.com/quay/claircore"
 	"github.com/quay/claircore/datastore/postgres"
+	"github.com/quay/claircore/indexer"
```

Comment:
we use `indexer` a lot in this file, so adding this import may cause confusion when someone sees `indexer` here. Can you rename this to `ccindexer`?